<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="srini-PC" ignored="0" name="Regression" tests="6" failures="6" timestamp="17 Jun 2020 07:07:47 GMT" time="36.063" errors="0">
  <testcase name="myTest" time="35.617" classname="testCases.Login_Testcase">
    <failure type="org.openqa.selenium.SessionNotCreatedException" message="Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find adb.exe in C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\platform-tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\emulator\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\bin\adb.exe. Do you have the Android SDK installed at &amp;apos;C:\Users\srini\AppData\Local\Android\Sdk\platform-tools&amp;apos;?
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;SRINI-PC&amp;apos;, ip: &amp;apos;192.168.1.6&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.8.0_171&amp;apos;
Driver info: driver.version: IntiateDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find adb.exe in C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\platform-tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\emulator\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\bin\adb.exe. Do you have the Android SDK installed at &amp;apos;C:\Users\srini\AppData\Local\Android\Sdk\platform-tools&amp;apos;?
    at getResponseForW3CError (C:\Users\srini\AppData\Local\Programs\Appium\resources\app\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Users\srini\AppData\Local\Programs\Appium\resources\app\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;SRINI-PC&amp;apos;, ip: &amp;apos;192.168.1.6&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.8.0_171&amp;apos;
Driver info: driver.version: IntiateDriver">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Could not find adb.exe in C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\platform-tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\emulator\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\bin\adb.exe. Do you have the Android SDK installed at 'C:\Users\srini\AppData\Local\Android\Sdk\platform-tools'?
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SRINI-PC', ip: '192.168.1.6', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_171'
Driver info: driver.version: IntiateDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find adb.exe in C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\platform-tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\emulator\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\bin\adb.exe. Do you have the Android SDK installed at 'C:\Users\srini\AppData\Local\Android\Sdk\platform-tools'?
    at getResponseForW3CError (C:\Users\srini\AppData\Local\Programs\Appium\resources\app\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Users\srini\AppData\Local\Programs\Appium\resources\app\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SRINI-PC', ip: '192.168.1.6', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_171'
Driver info: driver.version: IntiateDriver
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)
	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:336)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:88)
	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:98)
	at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:94)
	at com.ReaderManger.SeleniumFunLibrary.IntiateDriver.<init>(IntiateDriver.java:40)
	at testCases.Login_Testcase.myTest(Login_Testcase.java:29)
	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
	... 40 more
Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Could not find adb.exe in C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\platform-tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\emulator\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\bin\adb.exe. Do you have the Android SDK installed at 'C:\Users\srini\AppData\Local\Android\Sdk\platform-tools'?
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'SRINI-PC', ip: '192.168.1.6', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_171'
Driver info: driver.version: IntiateDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find adb.exe in C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\platform-tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\emulator\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\adb.exe,C:\Users\srini\AppData\Local\Android\Sdk\platform-tools\tools\bin\adb.exe. Do you have the Android SDK installed at 'C:\Users\srini\AppData\Local\Android\Sdk\platform-tools'?
    at getResponseForW3CError (C:\Users\srini\AppData\Local\Programs\Appium\resources\app\node_modules\appium-base-driver\lib\protocol\errors.js:826:9)
    at asyncHandler (C:\Users\srini\AppData\Local\Programs\Appium\resources\app\node_modules\appium-base-driver\lib\protocol\protocol.js:447:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
	at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
	at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
	at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
	at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	... 45 more
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
  <testcase name="myTest" time="0.018" classname="testCases.Login_Testcase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ReaderManger.BaseDriver.BaseDriver.<init>(BaseDriver.java:42)
	at pageobjects.OR_Login.<init>(OR_Login.java:17)
	at pages.Login.<init>(Login.java:23)
	at testCases.Login_Testcase.myTest(Login_Testcase.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
  <testcase name="myTest" time="0.001" classname="testCases.Login_Testcase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ReaderManger.BaseDriver.BaseDriver.<init>(BaseDriver.java:42)
	at pageobjects.OR_Login.<init>(OR_Login.java:17)
	at pages.Login.<init>(Login.java:23)
	at testCases.Login_Testcase.myTest(Login_Testcase.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
  <testcase name="myTest" time="0.001" classname="testCases.Login_Testcase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ReaderManger.BaseDriver.BaseDriver.<init>(BaseDriver.java:42)
	at pageobjects.OR_Login.<init>(OR_Login.java:17)
	at pages.Login.<init>(Login.java:23)
	at testCases.Login_Testcase.myTest(Login_Testcase.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
  <testcase name="myTest" time="0.001" classname="testCases.Login_Testcase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ReaderManger.BaseDriver.BaseDriver.<init>(BaseDriver.java:42)
	at pageobjects.OR_Login.<init>(OR_Login.java:17)
	at pages.Login.<init>(Login.java:23)
	at testCases.Login_Testcase.myTest(Login_Testcase.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
  <testcase name="myTest" time="0.001" classname="testCases.Login_Testcase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.ReaderManger.BaseDriver.BaseDriver.<init>(BaseDriver.java:42)
	at pageobjects.OR_Login.<init>(OR_Login.java:17)
	at pages.Login.<init>(Login.java:23)
	at testCases.Login_Testcase.myTest(Login_Testcase.java:33)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- myTest -->
</testsuite> <!-- Regression -->
